contracts/marketplace/MarketplaceInitFacet.sol

```diff
@@ -61,5 +63,10 @@ contract MarketplaceInitFacet is AMarketplaceFacet {
 
     // Initialize automatons storage.
     LibHasAutomatons.data().version = LibHasAutomatons.STORAGE_VERSION;
+
+    // ------------------------------------- //
+
+    // Initialize forwarder storage.
+    LibHasForwarder.data().forwarderAddress = address(0);
```

Comment:
Much more deterministic, I like it.
An auditor would tell you that it is un-necessary though, any storage variable is by default set to zero and wastes instructions.
I wouldn't remove it - and have kept zero-initialization instructions in init facets in the past as they document more than any comment.